benchmark/clblast/blas3/gemm.cpp

```diff
@@ -57,24 +66,25 @@ void BM_Gemm(benchmark::State& state) {
 
   // Create data
   // Scalars
-  scalar_t alpha = benchmark::utils::random_scalar<scalar_t>();
-  scalar_t beta = benchmark::utils::random_scalar<scalar_t>();
+  scalar_t alpha = blas_benchmark::utils::random_scalar<scalar_t>();
+  scalar_t beta = blas_benchmark::utils::random_scalar<scalar_t>();
 
   // Matrices
-  std::vector<scalar_t> a = benchmark::utils::random_data<scalar_t>(m * k);
-  std::vector<scalar_t> b = benchmark::utils::random_data<scalar_t>(k * n);
-  std::vector<scalar_t> c = benchmark::utils::const_data<scalar_t>(m * n, 0);
+  std::vector<scalar_t> a = blas_benchmark::utils::random_data<scalar_t>(m * k);
+  std::vector<scalar_t> b = blas_benchmark::utils::random_data<scalar_t>(k * n);
+  std::vector<scalar_t> c =
+      blas_benchmark::utils::const_data<scalar_t>(m * n, 0);
 
   // Specify the transpositions
-  clblast::Transpose a_tr = benchmark::utils::translate_transposition(t_a);
-  clblast::Transpose b_tr = benchmark::utils::translate_transposition(t_b);
+  clblast::Transpose a_tr = blas_benchmark::utils::translate_transposition(t_a);
+  clblast::Transpose b_tr = blas_benchmark::utils::translate_transposition(t_b);
 
   // Specify the layout. As with GEMV, this needs to be kColMajor, and results
   // in errors otherwise. It may be that this is incorrect (especially for
   // performance reasons), so may need to be revisited.
   auto layout = clblast::Layout::kColMajor;
 
-  ExecutorType* ex = Global::executorInstancePtr.get();
+  ExecutorType* ex = executorInstancePtr.get();
```

Comment:
Do we still need an executorInstancePtr when we're dynamically constructing the benchmarks? If so, then why is it no longer in a global namespace?

Reply:
No, I should pass it by reference. I will make the change

Reply:
Done